api/amqp.md

```diff
@@ -0,0 +1,222 @@
+# AMQP API
+
+This document specifies the messages expected by the connector through a publish/subscriber middleware that supports the AMPQ protocol. These messages can be listed in two main categories:
+
+### Southbound traffic (commands):
+
+- Update data
+- Request data
+
+### Northbound traffic (control, measurements):
+
+- Register thing
+- Unregister thing
+- Update thing's schema
+- Publish thing's data
+
+## Message types (Southbound)
+
+### Update data
+
+Updates a thing's sensor value.
+
+#### Topic
+
+* `fog`
```

Comment:
i think that the topic is `fog-messages`

Reply:
We assert the exchange only by using the `topic/exchange` name. The suffix `-messages` is only used when [asserting the message queue](https://github.com/CESARBR/knot-fog-connector/blob/master/src/network/AMQPConnection.js#L26) on message listening.

Reply:
Ack